scripts/ci_setup.py

```diff
@@ -386,6 +394,7 @@ def main(args: Any):
     pgo_config = ''
     physical_promotion_config = ''
     r2r_config = ''
+    experiment_config = ''
```

Comment:
It looks like we are missing the check here to see if we have an `experiment_config` set in the args and then setting this local variable, or am I missing something?

Reply:
That's next PR (for #3482).